src/sentry/snuba/metrics/extraction.py

```diff
@@ -80,16 +79,13 @@ class OnDemandMetricSpecVersioning:
     """
 
     spec_versions = [
-        SpecVersion(0),
-        SpecVersion(1, {"use_updated_env_logic"}),
+        SpecVersion(1),
```

Comment:
Do we want to always keep the versions with increasing integers also when we collapse them?

Reply:
The only thing that matters is that the integers are distinct once there's more than one element. I may want to enforce that or get rid of the version number altogether.

Reply:
In other words, there's no need to keep increasing it.

Reply:
Exactly, yeah, especially since it will always reflect the position of the version.